voice/methods/conferences/getConferences.md

```diff
@@ -0,0 +1,123 @@
+{% method %}
+
+## Retrieve Account's Conferences
+Returns a max of 1000 conferences, sorted by createdTime from oldest to newest
```

Comment:
Looking around the docs, looks like most places we write attributes like this. Also period.
```suggestion
Returns a max of 1000 conferences, sorted by `createdTime` from oldest to newest.
```